index.md

```diff
@@ -65,6 +65,7 @@ Many software programs already use `--color` or `-c` to enable color output. Thi
 | Software | Description | Date / Version Supported |
 |-|-|-|
 | [node](https://nodejs.org/node) | JavaScript runtime | |
+| [Nox](https://github.com/wntrblm/nox) | Flexible test automation | [2022-01-07 / 2022.1.7](https://github.com/wntrblm/nox/releases/tag/2022.1.7) |
 | [npm](https://www.npmjs.com/npm) | Package manager for JavaScript | |
 | [jest](https://jestjs.io/jest) | JavaScript testing framework | |
 | [nox](https://nox.thea.codes) | Python automation | [2022-01-07 / 2022.1.7](https://github.com/wntrblm/nox/releases/tag/2022.1.7) |
```

Comment:
I've already got nox in here, coule lines down.

Reply:
Oops, sorry for the duplicate! 

Let's capitalise the name, like the project does, and put them in alphabetical order.

I'll update the PR.